shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -78,7 +80,7 @@ public void createBridgeExecutor(ProcessorDTO processorDTO) {
                 .withName(BridgeExecutor.resolveResourceName(processorDTO.getId()))
                 .get();
 
-        if (existing == null || !expected.getSpec().equals(existing.getSpec())) {
+        if (createOrReplace(expected, existing)) {
```

Comment:
nitpick: `createOrReplace` is not very informative here. Maybe `isExecutorEqual`?

Reply:
Change no longer needed.